src/Compilers/Core/Portable/CodeAnalysisResources.Designer.cs

```diff
@@ -1343,7 +1343,7 @@ internal class CodeAnalysisResources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Windows PDB writer is not available -- could not find Microsoft.DiaSymReader.Native.{0}.dll.
+        ///   Looks up a localized string similar to Windows PDB writer doesn&apos;t support deterministic compilation: &apos;{0}&apos;.
```

Comment:
My guess is that someone forgot to run the generator, so I wound up with their changes?